### PR TITLE
Adding lidar package

### PR DESCRIPTION
**Pull request by giswqs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/lidar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0f6059a915c2f20668e74e347592171c3f897be90e69aea5ccf2c8e7707e1ec3
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (npavlovikj) [inline recipes/lidar/meta.yaml]: `noarch` recipes are built on Linux only, so you can ignore the Windows/OSX failures. Thus, this recipe can be `noarch: python` and you can remove the `skip` section. If some `run` dependencies are missing for Windows, the user can additionally install them, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

Comment (giswqs) [reply recipes/lidar/meta.yaml]: So I just revert to the previous commit?

Comment (npavlovikj) [reply recipes/lidar/meta.yaml]: Yes please.

File: recipes/lidar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=6.0
+    - matplotlib
```
Comment (npavlovikj) [inline recipes/lidar/meta.yaml]: If your package doesn't depend upon interactive images, but rather uses `matplotlib` to render images to files, then you should use `matplotlib-base` instead. `matplotlib` gives you the full Qt backend and consumes lots of space.

Comment (giswqs) [reply recipes/lidar/meta.yaml]: Thanks for the suggestion! I have changed it to `matplotlib-base`

**Comment by npavlovikj** [comment]
Thank you @giswqs !

**Pull request merged**